include/functions.h

```diff
@@ -237,9 +237,9 @@ void Effect_SpawnFragment(GlobalContext* globalCtx, Vec3f* burstDepthY, Vec3f* b
 // ? func_80029568(?);
 // ? func_80029724(?);
 // ? func_8002993C(?);
-// ? func_800299AC(?);
+void func_800299AC(GlobalContext* globalCtx, Vec3f* v);
 // ? func_80029C50(?);
-// ? func_80029CA4(?);
+void func_80029CA4(GlobalContext* globalCtx, s32 a, Vec3f* pos); //arg a1 is not s32, this will not OK func_8005E604
```

Comment:
ah. i probably assumed this was u8 because func_8005E604 passed in a u8. it's probably s32 because it results in non-matching codegen as u8, but i couldn't tell for sure